Assets/Scripts/SS3D/Systems/Screens/CameraFollow.cs

```diff
@@ -81,11 +81,32 @@ public class CameraFollow : Actor
         private const float SnapAngle = 45.1f;
 
         #endregion
-        
+
+        private void OnEnable()
+        {
+            try
```

Comment:
replace with` if(null && !didstart) return; else { Debug.Error(); return; }` So we can still have an error in case performed is null later in the game.